00-intro.md

```diff
@@ -64,13 +64,13 @@ Bash is the default shell on most modern implementations of Unix
 and in most packages that provide Unix-like tools for Windows.
 
 Using Bash or any other shell
-sometimes feels more like programming than like using a mouse.
+is more like programming than like using a mouse.
 Commands are terse (often only a couple of characters long),
 their names are frequently cryptic,
 and their output is lines of text rather than something visual like a graph.
 On the other hand,
 the shell allows us to combine existing tools in powerful ways with only a few keystrokes
-and to set up pipelines to handle large volumes of data automatically thus improving productivity and reproducibility.
```

Comment:
I also interpret that "pipeline" here refers to the use of `|`, rather than the use of shell scripts. In the pipes and filters lesson, we show how small tools like `wc` and `head` can be "combined" together to perform a complex task and potentially filter a large amount of data. I'm -1 for renaming to "workflow".